site/docs/contribute.md

```diff
@@ -45,6 +45,16 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. Before merging all review comments should be addressed either by making changes or agreeing the request is out of scope for the PR. For larger changes or additions to public APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in all ASF governed projects committers are expected to act in the best interest of the project. If a committer feels there might be a conflict of interest with a pull request they review, they are encouraged to ask for another committer to look at the pull request.
+
+A -1 vote on a PR (left as a comment with a reason attached) indicates a reviewer does not think the PR should be merged and that immediate changes by the PR author cannot address concerns. If the author and reviewer do not agree on the -1 vote, a discussion thread should be raised on the developer mailing list to come to a consensus on a path forward. One example where a -1 vote might be used is if a change should go through the [Iceberg improvement proposal](#apache-iceberg-improvement-proposals) process. Generally, -1 votes on PRs should be rare. Requesting changes on a PR indicates the reviewer believes the PR has merit but still needs issues addressed before merging. Reviewer in this context is anyone leaving comments on a PR, they can be a contributor, a committer or a PMC member.
```

Comment:
Would it make sense how a "-1 vote" should be expressed?
is it literally a comment with "-1", or "request changes" review option in github?

Reply:
I don't think `Request changes` should have the same weight as an actual `-1` on the PR. Also seems the text is clear on how the `-1` is expressed on the PR (via a comment with a reason)

Reply:
A PR is not strictly a vote, so I don't think -1 makes sense. If there's no consensus on a PR after requesting change, than, either the author close the PR, or he starts a vote on the dev mailing list. This vote would be consider like a code modification change, meaning veto is allowed, fragment vote is allowed as well.
https://www.apache.org/foundation/voting

Reply:
> I don't think `Request changes` should have the same weight as an actual `-1` on the PR.

it would definitely not be obvious to me.

the guidelines is important document, but we should assume people will remember them only partially, and be led by their intuition. In PR review context, `Request changes` is a common way of expressing necessity for changes on a PR. Having `Request changes` and "-1" comment meaning two different things will likely be confusing. At least it would be confusing to me.

Reply:
This can be solved by dropping the "-1" from this document, as proposed by @jbonofre in https://github.com/apache/iceberg/pull/10780#discussion_r1693000980

Reply:
I'll reword to remove -1

Reply:
reworded to avoid "-1"